tasks/lib/repositories.xml

```diff
@@ -1,253 +0,0 @@
-                    </fileset>
-                </delete>
-
-                <!-- Add a single placeholder file. -->
-                <touch file="${repo.dir}/.gitignore" />
-
-                <!-- The 'allFiles' flag on the GitCommitTask does not add files that aren't already tracked by git. -->
-                <exec command="git add --all" dir="${repo.dir}" />
-
-                <!-- Commit it all. -->
-                <gitcommit repository="${repo.dir}" message="Initial commit." allFiles="true" />
-            </then>
-            <else>
-                <!-- Otherwise just make sure the branch is checked out. -->
-                <gitcheckout repository="${repo.dir}" branchname="${repo.branch}" force="true" />
-
-                <!-- And pull, if there's a remote branch. -->
-                <if>
-                    <not><equals arg1="${branch_remote}" arg2="" /></not>
-                    <then>
-                        <trycatch>
-                            <try>
-                                <!--
-                                    Pull gently at first, allowing fast-forward commits but not obliterating
-                                    local commits. This allows us to build, examine, and rebuild without
-                                    pushing.
-
-                                    Ideally we would use Phing's git pull task, but this should fail if it's
-                                    not a fast forward, because we merges are not meaningful in the
-                                    artifact. If we DID use Phing's task, it would look like this:
-
-                                        <gitpull repository="${repo.dir}" refspec="${repo.branch}" />
-
-                                    Merges in the artifact are bad because they would create code that was
-                                    not generated directly from the development codebase; an artifact with
-                                    a merge would not be reproducible by re-running the artifact build task.
-                                    -->
-                                <exec command="git pull origin ${repo.branch} --ff-only" dir="${repo.dir}" checkreturn="true" />
-                            </try>
-                            <catch>
-                                <!--
-                                    Our local artifact has diverged from the upstream artifact. At this
-                                    point, we're going to lose our local commits, which is ok because this
-                                    is an artifact, not code under active development.
-                                    -->
-                                <gitpull repository="${repo.dir}" refspec="${repo.branch}:${repo.branch}" force="true" />
-                            </catch>
-                        </trycatch>
-                    </then>
-                </if>
-            </else>
-        </if>
-    </target>
-
-
-    <target name="commit">
```

Comment:
Oops! Looks like this target is still used by [artifact.xml](https://github.com/palantirnet/the-build/blob/21fd7c5625ffaaaf9d4c8ba0cb6b60eddc085953/tasks/artifact.xml#L269-L276). I better fix that!